### ISSUE TITLE
geoApplicationTest -> testSearchByGeocodeBasis -> AssertionError: logcount of GC2CJPF - Expecting actual not to be null

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#12667]
### Describe your problem!

New at CI - are there changes on the GC website ?
~~~
cgeo.geocaching.CgeoApplicationTest > testSearchByGeocodeBasis[android-30(AVD) - 11] [31mFAILED [0m
	java.lang.AssertionError: [logcount of GC2CJPF for type FOUND_IT]
	Expecting actual not to be null
~~~

### How to reproduce?

see eg. https://ci.cgeo.org/job/cgeo-CI_PR-build/1185/console

### Reproducible

Yes

### c:geo Version

CI tests

**Comment by Lineflyer** [comment]
Looks like it.
The log type count summary on the logbook tab is missing for me.

**Comment by Lineflyer** [comment]
To be precise:
c:geo keeps it on previously stored/opened caches, but if you open a unsaved cache the logbook summary is missing. I assume a parsing error.

**Comment by ztNFny** [comment]
fixed in https://github.com/cgeo/cgeo/pull/12670

**Comment by ztNFny** [comment]
CI fails but I don't understand why, see comment in https://github.com/cgeo/cgeo/pull/12670

**Comment by Lineflyer** [comment]
Fixed by #12670. Closing.
